src/Module/Configuration/Form/CarriersForm.php

```diff
@@ -502,6 +502,27 @@ static function ($value) use ($currency) {
                 'class'   => 'col-lg-2',
             ];
 
```

Comment:
Ik zou hier nog de insurancepossibilities voor BE ophalen, en deze hele fields[] toevoeging laten zitten als die er niet zijn (returns lege array als het goed is). Ook opletten wanneer het platform be is, dan hoef je dit helemaal niet te doen denk ik (anders krijg je local insurance poss. terug).

Reply:
Ok ik zie nu dat het verder ook zo is dat er dan een selectlijst met 1 optie (namelijk: 0) verschijnt...